physics/cu_gf_driver_post.F90

```diff
@@ -57,6 +65,21 @@ subroutine cu_gf_driver_post_run (im, t, q, prevst, prevsq, cactiv, cactiv_m, co
         else
           conv_act_m(i)=0.0
         endif
+        ! reflectivity parameterization for parameterized convection (reference:Unipost MDLFLD.f)
+        if(sqrt(garea(i)).lt.6500.)then
+        ze      = 0.0
+        ze_conv = 0.0
+        dbz_sum = 0.0
+        cuprate = raincv(i) * 3600.0 / dt          ! cu precip rate (mm/h)
+        ze_conv = 300.0 * cuprate**1.4
+        if (maxupmf(i).gt.0.05) then
+         do k = 1, km
+          ze = 10.0 ** (0.1 * refl_10cm(i,k))
+          dbz_sum = max(dbzmin, 10.0 * log10(ze + ze_conv))
```

Comment:
please change 10.0 to 10._kind_phys. (To avoid a potential precision demotion)

Reply:
@dustinswales Will do. Thanks.